serverless.yml

```diff
@@ -0,0 +1,20 @@
+service: mes-custom-authorizer
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+
+package:
+  individually: true
+
+functions:
+  authorization:
+    handler: src/functions/authorization/framework/handler.handler
```

Comment:
`handler.handler` is that the bast we can have? :)

Reply:
I thought for the journal we had `getJournal.handler` and thought that was the baseline so would expect `authoriser.handler` or sth but it's up to you

Reply:
Ah, the handler we specify in Terraform is subtly different. The only reason that the handler is specified in `serverless.yml` is to support running locally with serverless-offline (hence `handler.handler`). However, the artefacts of the build process are zip files containing `{{function-name}}.js` with a named export `handler`.